ui/apps/platform/src/Containers/Violations/ViolationsBreadcrumbs.tsx

```diff
@@ -1,17 +1,48 @@
+import React, { ReactElement } from 'react';
 import { Breadcrumb, BreadcrumbItem, Divider, PageSection } from '@patternfly/react-core';
+
 import BreadcrumbItemLink from 'Components/BreadcrumbItemLink';
-import React, { ReactElement } from 'react';
+import { FilteredWorkflowView } from 'Components/FilteredWorkflowViewSelector/types';
 
 import { violationsBasePath } from 'routePaths';
+import { ensureExhaustive } from 'utils/type.utils';
+
+function getTopLevelBreadcrumb(filteredWorkflowView: FilteredWorkflowView) {
+    switch (filteredWorkflowView) {
+        case 'Applications view':
+            return {
+                title: 'User workload violations',
+                url: `${violationsBasePath}?filteredWorkflowView=Application view`,
```

Comment:
Although not a blocker to merge, what are pro and con to lift these up to routePaths.ts file?
They also occur in HorizontalSubnav.tsx file.

Reply:
I think that makes sense - I'll make a note to adjust this since in this PR since it will still need a full CI run.